### PR TITLE
feat(portal): show "browser not supported" message

### PR DESCRIPTION
**Pull request by giac-mysten**
Closes: #91

### CODE REVIEW

File: portal/static/walrus-sites-portal-register-sw.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+function swNotLoadingNode() {
+    return titleSubtitleNode(
+        "Oh! Something's not right 🚧",
+        "Please try refreshing the page or unregistering the service worker."
+    );
+}
+
+function titleSubtitleNode(title, subtitle) {
+    let h3 = document.createElement("h3");
+    h3.textContent = title;
```
Comment (xydas97) [inline portal/static/walrus-sites-portal-register-sw.js:52]:
```suggestion
    h3.textContent = title;
    h3.className = "InterTightMedium";
```

File: portal/static/walrus-sites-portal-register-sw.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    let p = document.createElement("p");
+    p.textContent = subtitle;
```
Comment (xydas97) [inline portal/static/walrus-sites-portal-register-sw.js:54]:
```suggestion
    p.textContent = subtitle;
    p.className = "InterTightMedium";
    p.style.color = "#696969";
    p.style.fontSize = "18px";
```

**Review by xydas97 — CHANGES_REQUESTED** [review]
Minor styling changes:

image
image

**Comment by giac-mysten** [comment]
Thanks @xydas97 for the suggestions!

**Review by xydas97 — APPROVED** [review]
LGTM!

**Pull request merged**